content/pt-br/docs/tasks/configure-pod-container/configure-runasusername.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Configurando `RunAsUserName` Para Pods e Contêineres Windows
+content_type: task
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.18" state="stable" >}}
+
+Esta página mostra como usar a configuração `runAsUserName` para Pods 
+e contêineres que serão executados em nós Windows. Isso é aproximadamente 
+equivalente à configuração `runAsUser` específica do Linux, permitindo a você 
+executar aplicativos em um contêiner com um nome de usuário diferente do padrão. 
+
+## {{% heading "prerequisites" %}}
+
+Você precisa ter um cluster Kubernetes, e a ferramenta de linha de comando Kubectl
+deve ser configurada para se comunicar com o seu cluster. Espera-se que o cluster
+tenha nós `worker Windows`, onde os Pods com contêineres executando as cargas de trabalho do Windows,
+serão agendados.
+
+<!-- steps -->
+
+## Defina o nome de usuário para um Pod
+
+Para especificar o nome de usuário com o qual executar os processos de contêiner do Pod, 
+inclua o campo `securityContext` ([SecurityContext](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core)) 
```

Comment:
I'm a little confused with this SecurityContext link. The original docs point to the PodSecurityContext docs and have PodSecurityContext as text. What is the reason for changing it in the translation?